src/backend/joanie/core/filters/admin.py

```diff
@@ -23,6 +23,7 @@ class Meta:
         fields: List[str] = ["query"]
 
     query = filters.CharFilter(method="filter_by_query")
+    id = filters.AllValuesMultipleFilter(field_name="id")
```

Comment:
By convention we use plural form for filters which accepts several values. WDYT ?

```suggestion
    ids = filters.AllValuesMultipleFilter(field_name="id")
```